docs/peering.md

```diff
@@ -0,0 +1,197 @@
+# Peering
+
+## What is a peer / a peering relationship?
+
+The Interledger network is a graph of nodes (connectors) that have peered with one another by establishing a means of exchanging ILP packets and a means of paying one another for the successful forwarding and delivery of the packets.
+
+Rafiki includes an implementation of such a connector.
+
+In the Interledger protocol, connectors maintain peers, or counterparty connectors whom they transact with. A connector may extend a given peer a limited line of credit, or none at all, depending upon their trustworthiness.
+
+## What are the requirements for peering?
+
+Both counterparties need to
+
+- run an implementation of an Interledger connector, ideally Rafiki
+- agree on an asset of the peering relationship
+- agree on a `maxPacketAmount`, which specifies into how many packets a payment is split (or use default value within Rafiki)
+- communicate their static ILP address
+- communicate a connection endpoint for the peer to send packets to
+- exchange auth tokens for the connection endpoint
+
+Additionally, the two counterparties need to agree on a settlement mechanism. However, that is outside of the scope of Interledger / Rafiki.
+
+Note that two counterparties could have multiple peering relationships that differ in e.g. the underlying asset.
+
+## How to peer two Rafiki instances?
```

Comment:
Will this also refer to :point_down: when it's ready?
- https://github.com/interledger/rafiki/pull/969

Reply:
Yes. I didn't want to add it now since it does not exist yet. But I could mention that this is coming soon.